src/NHSD.BuyingCatalogue.Ordering.Domain/Order.cs

```diff
@@ -200,6 +200,16 @@ public Result Complete()
             return Result.Success();
         }
 
+        public bool DeleteOrderItem(CatalogueItemId catalogueItemId)
+        {
+            var orderItem = orderItems.SingleOrDefault(x => x.CatalogueItem.Id == catalogueItemId);
```

Comment:
As above with the lambda parameter name.

Reply:
done